go-controller/pkg/ovn/acl.go

```diff
@@ -2,12 +2,64 @@ package ovn
 
 import (
 	"fmt"
+	knet "k8s.io/api/networking/v1"
+	"strings"
+
 	libovsdbclient "github.com/ovn-org/libovsdb/client"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/libovsdbops"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
 )
 
+func joinACLName(substrings ...string) string {
+	return strings.Join(substrings, "_")
+}
+
+type ACLType string
+
+const (
+	ACLTypeIngress ACLType = "Ingress"
+	ACLTypeEgress  ACLType = "Egress"
+)
+
+func (aclType ACLType) ToPolicyType() knet.PolicyType {
```

Comment:
If we have this, we should have a `FromPolicyType`as well.
Probably free methods, like `aclTypeToPolicyType` and `policyTypeToACLType`.